apps/neuvector/neuvector/ithc/00.yaml

```diff
@@ -12,11 +12,6 @@ spec:
           shareName: neuvector-data-00
         ingress:
           host: cft-neuvector00-api.ithc.platform.hmcts.net
-          path: "/"
-          enabled: true
-          annotations:
-            kubernetes.io/ingress.class: traefik
-            traefik.ingress.kubernetes.io/router.tls: "true"
         configmap:
           data:
             sysinitcfg.yaml: |
```

Comment:
somehow Kustomize would not merge `.configmap.data.*.yaml` configs defined in base and overlays